src/main/kotlin/com/avail/descriptor/functions/CompiledCodeDescriptor.kt

```diff
@@ -1005,7 +1005,7 @@ class CompiledCodeDescriptor private constructor(
 			Int,
 			A_Tuple,
 			A_Phrase"""))
-	override fun mutable(): AbstractDescriptor = throw unsupportedOperation()
+	override fun mutable() = throw unsupportedOperation()
```

Comment:
It looks like `throw unsupportedOperation()` does the same thing as simply `unsupportedOperation()`, and that the `Nothing` type allows the first form to continue to be type correct.  After Kotlinization is complete, we should change them all to the latter form.  We should also remove `unsupportedOperationException()`, and move `unsupportedOperation` into a companion of `AvailUnsupportedOperationException`.  For now, let's just let it be.

Reply:
I systematically cleaned these.